lib/Pingmachine/Probe/PPing.pm

```diff
@@ -0,0 +1,257 @@
+package Pingmachine::Probe::PPing;
+
+use Any::Moose;
+use AnyEvent;
+use AnyEvent::Util;
+use Log::Any qw($log);
+use List::Util qw(shuffle);
+
+my $SCION_DIR = $ENV{'SC'} // '/opt/OSAGscion';
```

Comment:
The default should be as in the upstream project and not as in a internal package name.

Reply:
Done